<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scout</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>223</width>
      <height>323</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>219</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>219</width>
         <height>81</height>
        </size>
       </property>
       <property name="title">
        <string>Timers</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>21</y>
          <width>201</width>
          <height>55</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_armes">
             <property name="text">
              <string>Reset armes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>En cours ..</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UP 100</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UP 150</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Camp buff</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_timer_armes">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;35:00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_timer_camp">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;05:00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>219</width>
         <height>107</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>219</width>
         <height>107</height>
        </size>
       </property>
       <property name="title">
        <string>Simulation</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>23</y>
          <width>201</width>
          <height>83</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="comboBox_Type">
             <item>
              <property name="text">
               <string>Tour</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Fort</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ch√¢teau</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_Ravit">
             <item>
              <property name="text">
               <string>T1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>T2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>T3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Canons </string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mortiers</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TP</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="checkBox_double">
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_1Dolyak">
             <property name="text">
              <string>Dolyak simple</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_dolyaks">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;x/x&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_log">
       <property name="maximumSize">
        <size>
         <width>221</width>
         <height>121</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuAlarme_sonore">
     <property name="title">
      <string>Alarme sonore</string>
     </property>
     <addaction name="actionAucune"/>
     <addaction name="actionCoucou"/>
     <addaction name="actionDring"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionPop_up_reset_des_armes"/>
    <addaction name="menuAlarme_sonore"/>
   </widget>
   <widget class="QMenu" name="menuLog">
    <property name="title">
     <string>Log</string>
    </property>
    <addaction name="actionVider_le_log"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="actionCredits"/>
   </widget>
   <addaction name="menuAffichage"/>
   <addaction name="menuLog"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionTimers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Timers</string>
   </property>
  </action>
  <action name="actionSimulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="actionPop_up_reset_des_armes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pop-up reset des armes</string>
   </property>
  </action>
  <action name="actionVider_le_log">
   <property name="text">
    <string>Vider le log</string>
   </property>
  </action>
  <action name="actionEnregistrer_le_log">
   <property name="text">
    <string>Enregistrer le log</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Aide..</string>
   </property>
  </action>
  <action name="actionPersonnaliser">
   <property name="text">
    <string>Personnaliser ..</string>
   </property>
  </action>
  <action name="actionCredits">
   <property name="text">
    <string>√Ä propos..</string>
   </property>
  </action>
  <action name="actionAucune">
   <property name="text">
    <string>Aucune </string>
   </property>
  </action>
  <action name="actionCoucou">
   <property name="text">
    <string>Coucou</string>
   </property>
  </action>
  <action name="actionDring">
   <property name="text">
    <string>Dring</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
